### ISSUE TITLE
Unable to launch debug session for Flash SWF on Linux

### ISSUE THREAD

**Issue by ba0f3** [BowlerHatLLC/vscode-swf-debug#1]
After recent updates, I can't use `launch` debug session for SWF on Linux anymore.

Here my `launch.json`:
```json
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "swf",
      "request": "attach",
      "name": "Attach SWF",
    },
    {
        "type": "swf",
        "request": "launch",
        "name": "Launch SWF",
        "runtimeExecutable": "/usr/bin/flashplayerdebugger",
        "program": "out/index.swf"
    }
  ]
}
```
It says: "*Error launching SWF debug session.*" even I run with `runtimeExecutable` param or not (I made it default program for the .swf). `attach` method still works fine, but I have to run `flashplayerdebugger` from terminal manually

**Comment by joshtynjala** [comment]
According to my notes, I've tried the _libflashplayer.so_ plugin in Firefox on Linux (and I remember that it worked, but was unstable), but I don't think that I've tried the standalone _flashplayerdebugger_ executable. You might be the only person who has!

Can you run **Toggle Developer Tools** command in VSCode after it fails to see if a more detailed error message appears in the console?

Also, one thing that you might try is launching _${workspaceFolder}/out/index.swf_. This should pass in the absolute path instead of a relative path, which may be better.

**Comment by ba0f3** [comment]
I used to debug with `flashplayerdebugger` before, hadnt try to debug using Firefox yet. and there was no error in developer tools

**Comment by joshtynjala** [comment]
@ba0f3 I just pushed a commit that includes some extra information to the "Error launching SWF debug session" message. It will be included in the next update, and hopefully, it will give us some more clues. I'll let you know when I release the update.

**Comment by joshtynjala** [comment]
@ba0f3 I just published vscode-swf-debug v1.0.2. It includes a fix for launching the standalone flashplayerdebugger executable on Linux, and I confirmed it is working correctly on my Linux computer running Xubuntu. Please let me know if it helps. Thanks!

**Comment by ba0f3** [comment]
it works again, thank you!
